api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupTxCurr.java

```diff
@@ -84,7 +84,7 @@ public ReportDefinition constructReportDefinition() {
 	public List<ReportDesign> constructReportDesigns(ReportDefinition reportDefinition) {
 		ReportDesign reportDesign = null;
 		try {
-			reportDesign = createRowPerPatientXlsOverviewReportDesign(reportDefinition, "TXCURR.xls", "TXCURR.xls_", getExcelDesignUuid(), null);
+			reportDesign = createXlsReportDesign(reportDefinition, "TXCURR.xls", "TXCURR.xls_", getExcelDesignUuid(), null);
```

Comment:
Renaming this method to a general purpose is ok, but after mearging your code will need to refactor all other places that uses the method

Reply:
Yup, @nthfloor will have to take this into account too when merging the code to master.

Reply:
Yes this is assumed, we are likely going to have to resolve merge conflicts across a number of files including Sql and composition cohort queries but yes good to note.